crates/uv/tests/it/build.rs

```diff
@@ -898,7 +898,7 @@ fn build_constraints() -> Result<()> {
 
 #[test]
 fn build_sha() -> Result<()> {
-    let context = TestContext::new("3.8");
+    let context = TestContext::new("3.9");
```

Comment:
any reason no to bump those to 3.12?

Reply:
Most things I didn't bump to 3.12 failed on that version, let me double check this one though..